src/java_binding/src/lib.rs

```diff
@@ -552,6 +592,90 @@ pub extern "system" fn Java_com_risingwave_java_binding_Binding_rowGetDecimalVal
     })
 }
 
+#[no_mangle]
+pub extern "system" fn Java_com_risingwave_java_binding_Binding_rowGetDateValue<'a>(
+    env: EnvParam<'a>,
+    pointer: Pointer<'a, JavaBindingRow>,
+    idx: jint,
+) -> JObject<'a> {
+    execute_and_catch(env, move || {
+        let value = pointer
+            .as_ref()
+            .datum_at(idx as usize)
+            .unwrap()
+            .into_date()
+            .0
+            .to_string();
+
+        let string_value = env.new_string(value)?;
+        let (class_ref, constructor) =
+            pointer.as_ref().class_cache.date_ctor.get_or_try_init(|| {
+                let cls = env.find_class("java/sql/Date")?;
```

Comment:
Is is possible to create a `Data` and `Time` object from a integer timestamp instead of from string?

Reply:
I think for `Date` and `Time`, their minimal required precision is millisecond, and therefore there is no need to care about the nano second level precision. And I think it should be safe to use the constructor from a timestamp of long type. It takes more time to convert to a string first and then parse the string. 

cc @WillyKidd 

https://docs.oracle.com/javase/8/docs/api/java/sql/Time.html#Time-long-
https://docs.oracle.com/javase/8/docs/api/java/sql/Date.html#Date-long-

Reply:
maybe we can do this in the next pr :)